astropy/io/ascii/ui.py

```diff
@@ -412,6 +437,7 @@ def _get_guess_kwargs_list(read_kwargs):
     # FixedWidthTwoLine would also be read by Basic, so it needs to come first.
     if len(read_kwargs) > 0:
         for reader in [fixedwidth.FixedWidthTwoLine,
+                       fastbasic.FastBasic,
```

Comment:
@hamogu - in practice `len(read_kwargs) > 0` is almost always true because of the default fill values.  Therefore this list should match the list below, hence the addition of `FastBasic` here.